asminc/cbm_kernal.inc

```diff
@@ -1,7 +1,7 @@
 ;
 ; Olli Savia <ops@iki.fi>
 ;
-; Commodore kernal functions
+; Commodore kernal functions http://web.archive.org/web/20180720093634/https://www.pagetable.com/?p=926
```

Comment:
FWIW - while I hold the author of "pagetable" in high esteem, I don't see putting such links here as something adding value.

Reply:
I know what you mean but you won't find a more complete list.